src/control/server/config.go

```diff
@@ -108,7 +108,8 @@ func loadConfigOpts(cliOpts *cliOptions, host string) (
 	log.Debugf("DAOS config read from %s", config.Path)
 
 	// get unique identifier to activate SPDK multiprocess mode
-	config.NvmeShmID = hash(host + strconv.Itoa(os.Getpid()))
+	// TODO: re-enable
+	//config.NvmeShmID = hash(host + strconv.Itoa(os.Getpid()))
```

Comment:
Is this change related to the purpose of this patch?

Reply:
related but experimental only, removed